core/test/nuxeo-connection.test.js

```diff
@@ -14,7 +14,7 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
  */
-import { fixture, html } from './test-helpers';
+import { fixture, html } from '@nuxeo/nuxeo-testing-helpers/test-helpers';
```

Comment:
let's add main to this package (probably just introduce an index.js) so we just `import { fixture, html } from '@nuxeo/nuxeo-testing-helpers'`